app/controllers/spina/registers_controller.rb

```diff
@@ -71,10 +71,32 @@ def history
 
         { current_record: current_record, previous_record: previous_record, updated_fields: changed_fields, key: entry[:key] }
       }
+
+      filter(params[:search_input]) unless params[:search_input].nil?
```

Comment:
can we call this param `q` to be consistent with the search on the records page?

Reply:
`q` looks a little bit meaningless (I know it stands for query, but...), if we agree in something with more meaning I can change it